### PR TITLE
Implement a Food ingredients explorer.

### PR DESCRIPTION
**Pull request by n1k0**
[User Story](https://www.notion.so/Initier-un-explorateur-d-ingr-dients-4a39cf204de145c9a0f81d03c096e493)

### CODE REVIEW

File: src/Page/Explore.elm [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    | FoodDbLoaded (WebData BuilderDb.Db)
+
+
+init : Scope -> Dataset -> Session -> ( Model, Session, Cmd Msg )
+init scope dataset session =
+    ( { dataset = dataset, scope = scope }
+    , session
+    , Cmd.batch
+        [ if BuilderDb.isEmpty session.builderDb then
+            FoodRequestDb.loadDb session FoodDbLoaded
```
Comment (magopian) [inline src/Page/Explore.elm:58]: Do we want to only load the food db if the scope is `Food`?

**Review by magopian — APPROVED** [review]
This looks very nice. There's the issue of the "url encoding" of IDs with spaces... we might want to remove the spaces in the food ingredients, or add url encoding (if we ever forget and let spaces sneak back in, I guess it's good practice).

There's a [PR that merges the ecobalyse-data repo](https://github.com/MTES-MCT/ecobalyse/pull/207) that'll make it easier to add changes to the json files in a PR, in case we want to merge it first?

File: src/Page/Explore.elm (hunk 1 above)
Comment (n1k0) [reply src/Page/Explore.elm:58]: done in https://github.com/MTES-MCT/ecobalyse/pull/208/commits/1a997da208ece37fed202c4e1655e5b2ff033e0f

**Comment by n1k0** [comment]
> or add url encoding (if we ever forget and let spaces sneak back in, I guess it's good practice).

If you look carefully it's done in https://github.com/MTES-MCT/ecobalyse/pull/208/commits/e4ec856932fce17ed01407ad41936c91bc814cb3

**Comment by n1k0** [comment]
I'm landing this, we can update ingredients id later in another PR.

**Comment by magopian** [comment]
> If you look carefully it's done in https://github.com/MTES-MCT/ecobalyse/commit/e4ec856932fce17ed01407ad41936c91bc814cb3

Ah yes indeed, I missed that! Well, perfect then 👍

**Pull request merged**